thrift/src/main/thrift/atoms/media.thrift

```diff
@@ -0,0 +1,27 @@
+namespace * contentatom.media
+namespace java com.gu.contentatom.thrift.atom.media
+#@namespace scala com.gu.contentatom.thrift.atom.media
+
+typedef i64 Version
+
+enum AssetType {
+  YOUTUBE,
+  FACEBOOK,
+  PODCAST,
+  DAILYMOTION,
+  MAINSTREAM,
```

Comment:
@paulmr @fredex42 I find this list confusing because we mix asset `type` (`PODCAST` which I think should be `AUDIO`), video platform providers (`YOUTUBE`, `FACEBOOK`, `DAILYMOTION`) and syndication partners (`MAINSTREAM`). I think it will be better to be closer to the current asset model in composer where we have `types` (`audio`, `video`) and `source` (`youtube`, `guardian`, etc..)